src/core/instance.cc

```diff
@@ -56,10 +56,6 @@ Instance::Instance(const std::vector<std::string>& args) try
     : m_Exception(__func__),
       m_Config(args)
   {
-    // TODO(anonimal): Initializing of sources/streams/sinks must come after we've properly configured the logger.
-    //   we do this here so we can catch debug logging before instance "initialization". This is not ideal
```

Comment:
Please don't remove this TODO. It is not yet resolved. You can move it along with the function.